datadog_checks_base/datadog_checks/data/agent_requirements.in

```diff
@@ -39,14 +39,15 @@ pysnmp==4.4.3
 pysnmp-mibs==0.1.6
 python-binary-memcached==0.26.1; sys_platform != 'win32'
 pyvmomi==v6.5.0.2017.5-1
+pywin32==224; sys_platform == 'win32'
 redis==2.10.5
 requests==2.19.1
 requests_ntlm==1.1.0
 scandir==1.8
+selectors34==1.2.0; sys_platform == 'win32' and python_version < '3.4'
 serpent==1.27; sys_platform == 'win32'
 simplejson==3.6.5
 six==1.11.0
 supervisor==3.3.3; python_version < '3.0'
-testfixtures==4.14.3; sys_platform == 'win32'
```

Comment:
Do you know when/where this was removed, just want to confirm its intentional

Reply:
https://github.com/DataDog/integrations-core/pull/2322/files#diff-a140a1a4b2a0cc3562a34cffa664df44